charts/dev/livebook/values.yaml

```
@@ -50,3 +51,15 @@ workload:
               secretKeyRef:
                 name: secrets
                 key: LIVEBOOK_COOKIE
+            LIVEBOOK_PASSWORD:
+              secretKeyRef:
+                name: secrets
+                key: LIVEBOOK_PASSWORD
+            LIVEBOOK_DEBUG: '{{ ternary "true" "false" .Values.livebookDebug }}'
+            LIVEBOOK_UPDATE_INSTRUCTIONS_URL: '{{ .Values.livebookUpdateInstructionsUrl }}'
+            LIVEBOOK_AWS_CREDENTIALS: '{{ ternary "true" "false" .Values.livebookAwsCredentials }}'
+            AWS_ACCESS_KEY_ID: '{{ .Values.awsAccessKeyId }}'
+            AWS_SECRET_ACCESS_KEY:
+              secretKeyRef:
+                name: secrets
+                key: AWS_SECRET_ACCESS_KEY
```

```suggestion
            LIVEBOOK_PASSWORD:
              secretKeyRef:
                name: secrets
                key: LIVEBOOK_PASSWORD
            LIVEBOOK_DEBUG: '{{ ternary "true" "false" .Values.livebookDebug }}'
            LIVEBOOK_UPDATE_INSTRUCTIONS_URL: "{{ .Values.livebookUpdateInstructionsUrl }}"
            LIVEBOOK_AWS_CREDENTIALS: '{{ ternary "true" "false" .Values.livebookAwsCredentials }}'
            AWS_ACCESS_KEY_ID: "{{ .Values.awsAccessKeyId }}"
            AWS_SECRET_ACCESS_KEY:
              secretKeyRef:
                name: secrets
                key: AWS_SECRET_ACCESS_KEY
```

Lets only use squote when there are dquotes inside!